static/app/components/forms/formPanel.tsx

```diff
@@ -55,14 +56,17 @@ function FormPanel({
   renderFooter,
   renderHeader,
   collapsible,
+  id,
   initiallyCollapsed = false,
   ...otherProps
 }: Props) {
   const [collapsed, setCollapse] = useState(initiallyCollapsed);
   const handleCollapseToggle = useCallback(() => setCollapse(current => !current), []);
 
   return (
-    <Panel id={typeof title === 'string' ? sanitizeQuerySelector(title) : undefined}>
+    <Panel
+      id={id ?? (typeof title === 'string' ? sanitizeQuerySelector(title) : undefined)}
+    >
```

Comment:
Instead of `id` can we perhaps use the sanitized title here? Since we already have a standard way of making a panel id?

Reply:
Used title instead.